packages/fluentui/e2e/tests/dropdownMoveFocusOnTab-test.ts

```diff
@@ -29,5 +30,15 @@ describe('Dropdown', () => {
       expect(await e2e.isFocused(triggerButton)).toBe(false);
       expect(await e2e.isFocused(previousFocusableSibling)).toBe(true);
     });
+
+    it('closes dropdown on outside click', async () => {
+      await e2e.clickOn(triggerButton);
+
+      expect(await e2e.count(listItem)).toBe(inputItems.length);
```

Comment:
```suggestion
      expect(await e2e.count(listItem)).toBeGreaterThan(0);
```

In this case we don't need to have `inputItems` exported

Reply:
Interesting 😀although I am tempted to leave it like this, just in case the example will have some loading / no results indicators added at some point (those count as dropdown items) and we may get a false positive. I think it's more robust like this.